python/CHANGELOG.rst

```diff
@@ -1,3 +1,13 @@
+--------------------
```

Comment:
Sorry, seems I forgot the post-release for 0.3.4!

Reply:
No worries - I had mucked up your workflow by setting the release number as part of a PR!